brave-lists/debounce.json

```diff
@@ -428,6 +428,15 @@
     "action": "regex-path",
     "param": "^/([^/]+)/s(/.*)$"
   },
+  {
+    "include": [
+      "*://shopping.yahoo.com/rdlw?*"
+    ],
+    "exclude": [
+    ],
+    "action": "redirect",
+    "param": "gcReferrer"
```

Comment:
The name is suspicious, but it does seem to be the final destination.

Reply:
As seen on https://www.engadget.com/computing/laptops/best-laptops-120008636.html